airflow/providers/celery/provider.yaml

```
@@ -32,7 +32,6 @@ versions:
 
 additional-dependencies:
   - apache-airflow>=2.1.0
```

```suggestion
  - apache-airflow>=2.2.0
```

This is probably what we want, as 2.2.0 was the first version with celery 5?

---

Right

---

Though I think this ship has sailed (unless we also yank recent celery provider releases)

---

I think celery **COULD** be held back by other dependencies for Airlflow pre 2.2.0 - I will check it shortly.